files/chef-marketplace-cookbooks/chef-marketplace/templates/default/delivery.rb.erb

```diff
@@ -12,7 +12,9 @@ delivery["chef_server"]           = "https://localhost:8443/organizations/delive
 delivery["chef_server_proxy"]     = true
 delivery["chef_username"]         = "delivery"
 delivery["primary"]               = true
+<% unless node['chef-marketplace']['platform'] == 'azure' -%>
 delivery["marketplace_licensing"] = true
```

Comment:
perhaps the boolean here could be triggered by the value instead of just not rendering it?